cranelift/codegen/src/prelude.isle

```diff
@@ -369,15 +357,15 @@
 (decl int_bool_fits_in_32 (Type) Type)
 (extern extractor int_bool_fits_in_32 int_bool_fits_in_32)
 
-;; An extractor that matches I64 or B64.
+;; An extractor that matches I64.
```

Comment:
We can probably remove this extractor now and just use a constant `$I64`, no? That has the nice extra benefit of allowing more precise overlap checking...

Reply:
That's a great point! I hadn't started thinking about improvements that we could make without the boolean types, and was just in "make it compile" mode instead :)

Reply:
I've made this change, and removed/refactored similar extractors in the prelude.